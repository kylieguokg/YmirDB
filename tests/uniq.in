uniq a
set a
get a
uniq a
get a
append a 1
get a
uniq a
get a
set a 1 2 3
get a
uniq a
get a
set a 1 1 1
get a
uniq a
get a
set b 1 2 3 2 1
get b
uniq b
get b
set c 1 2 2 3 3 3 4 4 4 4 5 5 5 5 5
get c
uniq c
get c
set w0w 1 0 0 1 0 0 1
get w0w
uniq w0w
get w0w
bye
