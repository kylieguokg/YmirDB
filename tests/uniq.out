> no such key

> ok

> []

> ok

> []

> ok

> [1]

> ok

> [1]

> ok

> [1 2 3]

> ok

> [1 2 3]

> ok

> [1 1 1]

> ok

> [1]

> ok

> [1 2 3 2 1]

> ok

> [1 2 3 2 1]

> ok

> [1 2 2 3 3 3 4 4 4 4 5 5 5 5 5]

> ok

> [1 2 3 4 5]

> ok

> [1 0 0 1 0 0 1]

> ok

> [1 0 1 0 1]

> bye
